ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -747,11 +747,11 @@ public void removeListeners(double count) {}
         return null;
       }
       String headerName = HeaderUtil.stripHeaderName(header.getString(0));
-      String headerValue = HeaderUtil.stripHeaderValue(header.getString(1));
+      String headerValue = header.getString(1);
       if (headerName == null || headerValue == null) {
         return null;
       }
-      headersBuilder.add(headerName, headerValue);
+      headersBuilder.addUnsafeNonAscii(headerName, headerValue);
```

Comment:
We have internal failures due to this line. The internal system is not able to find it.
I'm not an Android expert and I couldn't find from which version of OkHTTP this is available. 
Could you help me out a bit here? :pray:

Reply:
It was added in 3.12.0: https://square.github.io/okhttp/changelogs/changelog_3x/#version-3120
RN is using v4 of okhttp now: https://github.com/facebook/react-native/blob/5fda72f678fd656daaa3b988410f3b3a8701b7e4/ReactAndroid/gradle.properties#L17

Reply:
Hi @cipolleschi is there any update on this?

All non-internal builds passed so I suppose the code should have built without problems.

Is the internal build using a different okhttp version than the one on GitHub? If you could attach logs (without sensitive info of course), it would be easier for me to help.

Also, I saw the Linter complained about a warning (not an error). Does this mean it is still mergeable?

Reply:
Jumping on this.

The build still fails internally. For your reference with this error:

```
ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java:754: error: cannot find symbol
[CONTEXT]       headersBuilder.addUnsafeNonAscii(headerName, headerValue);
[CONTEXT]                     ^
[CONTEXT]   symbol:   method addUnsafeNonAscii(java.lang.String,java.lang.String)
[CONTEXT]   location: variable headersBuilder of type okhttp3.Headers.Builder
```

> Also, I saw the Linter complained about a warning (not an error). Does this mean it is still mergeable?

Currenty we can't merge this as it is, as our internal builds are red (not sure why they don't show up on Github though).

Reply:
Thank you @cortinico for looking into this!

So the method couldn't be found when compiled...

The code builds fine on CircleCI though. Is there any particularly special setup for internal builds?

For example, was the build that failed configured with a lower version of OkHttp?

Since I don't have access to the build logs and configurations, what's the best approach to the issue if the code builds fine in public but fails internally?

Thanks!